lib/jobs/ansible-job.js

```diff
@@ -78,10 +95,23 @@ function ansibleJobFactory(
      * @memberOf AnsibleJob
      */
      AnsibleJob.prototype._cleanup = function cleanup() {
+        var self = this;
+        try {
+            var data = fs.readFileSync(self.outFileName, 'utf8');
```

Comment:
I'll move code into _run, that looks better.  And I'm not sure I understand the 1st part of your comment (the 'context.ansibleResultFile' var is initialized up front - is that enough?